docs/user-guide/binned-data/filtering.ipynb

```diff
@@ -43,28 +126,17 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import numpy as np\n",
-    "import scipp as sc"
+    "import scipp as sc\n",
+    "\n",
+    "da = sc.data.vulcan_steel_strain_data()\n",
+    "da"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "np.random.seed(1) # Fixed for reproducibility\n",
-    "end_time = 100000\n",
-    "tof_max = 10000\n",
-    "width = tof_max/20\n",
-    "sizes = 4*np.array([7000, 3333, 3000, 5000])\n",
-    "size = np.sum(sizes)\n",
-    "data = sc.ones(dims=['event'], unit='counts', shape=[size], with_variances=True)\n",
-    "time = sc.zeros(dims=['event'], unit='s', dtype='datetime64', shape=[size])\n",
-    "# time-of-flight in a neutron-scattering experiment\n",
-    "tof = sc.zeros(dims=['event'], unit='us', dtype='float64', shape=[size])\n",
-    "table = sc.DataArray(data=data, coords={'time':time, 'tof':tof})\n",
-    "table"
+    "The `dspacing` dimension is the interplanar lattice spacing, and plotting this data we see two diffraction peaks:"
```

Comment:
I doubt anyone who isn't a physicist or chemist will know what an interplanar lattice spacing is. Could say 'spacing between planes in a crystal'. Seems more intuitive to me.